spec/mail/mailbox.mdown

```diff
@@ -47,6 +68,23 @@ A **Mailbox** object has the following properties:
   The user may rename the mailbox or make it a child of another mailbox. This property is server-set.
 - **mayDelete**: `Boolean`
   The user may delete the mailbox itself. This property is server-set.
+- **namespace**: `Namespace`
+  Describes if this mailbox is a personal mailbox or a shared mailbox.
```

Comment:
IMO we should make this more generic.

This gives additional information about the mailbox. I don't see why it would be restricted to "delegation".

We can imagine the following types: personal, delegated, shared, external...

Maybe we can say:

```
Gives additional information about the type of this mailbox. This can now take the values 'personal' or 'delegated'.
```

And then allow people to define their extensions of the *namespace* object. WDYT?

Reply:
ok